bubblewrap.c

```diff
@@ -181,7 +182,7 @@ lock_file_new (const char *path)
 static void
 usage (int ecode, FILE *out)
 {
-  fprintf (out, "usage: %s [OPTIONS...] COMMAND [ARGS...]\n\n", argv0);
+  fprintf (out, "usage: %s [OPTIONS...] --exec-filename COMMAND [ARGS...]\n\n", argv0);
```

Comment:
If the synopsis line listed every option, it would quickly get unmanageably long. Just treat this new option as part of `OPTIONS`.

(Also, it should be `[--exec-filename PATH]` if you were going to do it like this, because it takes an argument and is (or should be) optional.)

Reply:
So, I made a huge confusion here. In my mind, --exec-filename was meant to change the argv[0]. Having a second look at #91, it's not that he want here. A new version is ready, just writing a test for it to ensure it's working.

Thanks for the patience!